server.js

```diff
@@ -73,7 +73,7 @@ app.use(bodyParser.json());
 app.use(cookieParser());
 app.use(cors(
     {
-        origin: ["http://localhost:2000", "http://localhost:3000","http://localhost:80/","http://digitalbraininjury.com/"],
+    origin: ["http://localhost:2000", "http://localhost:3000","http://localhost:80/","http://digitalbraininjury.com","http://www.digitalbraininjury.com"],
```

Comment:
can the domain made into a variable and passed in from configuration file? Soon we will be moving to nsfcareer.io